src/__swaps__/screens/Swap/components/SwapInputValuesCaret.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, useColorMode } from '@/design-system';
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import Animated, { Easing, SharedValue, useAnimatedStyle, withRepeat, withSequence, withTiming } from 'react-native-reanimated';
+import { SLIDER_COLLAPSED_HEIGHT, SLIDER_HEIGHT, caretConfig } from '@/__swaps__/screens/Swap/constants';
+import { equalWorklet } from '@/__swaps__/safe-math/SafeMath';
+import { NavigationSteps } from '@/__swaps__/screens/Swap/hooks/useSwapNavigation';
+import { useSwapContext } from '@/__swaps__/screens/Swap/providers/swap-provider';
+import { inputKeys } from '@/__swaps__/types/swap';
+import { getColorValueForThemeWorklet } from '@/__swaps__/utils/swaps';
+
+export function SwapInputValuesCaret({ inputCaretType, disabled }: { inputCaretType: inputKeys; disabled?: SharedValue<boolean> }) {
```

Comment:
new component for the caret that is used for the input amount, output amount, input native value, and output native value